src/introduction.md

```diff
@@ -10,6 +10,15 @@ For tools and libraries relating to Rust on ESP, please see the [esp-rs organiza
 [espressif]: https://espressif.com/
 [esp-rs organization]: https://github.com/esp-rs/
 
+## Status of This Book
+
+This book is currently a work in progress. A number of sections may be missing information or be missing altogether. If there is a specific topic you would like to see documented please [open an issue].
+
+If you feel you can contribute something to this book, we encourage you to [create a pull request]!
+
+[open an issue]: https://github.com/esp-rs/book/issues
```

Comment:
Nit: maybe link to https://github.com/esp-rs/book/issues/new instead, to save a click?